OxfordBodleian/Juel-Jensen/BDLaethd17.xml

```diff
@@ -160,63 +401,170 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="left">17</dim>
                                         <dim type="intercolumn">12</dim>
                                     </dimensions>
+                                    
+                                    <note> The characters per line are 10–12. </note>
+                                    
+                                    <ab type="pricking">Ruling is visible; pricking is
+                                        visible.</ab>
+                                    <ab type="ruling" subtype="pattern">
+                                        <locus from="3ra" to="139vb"/> Ruling pattern:
+                                        1A-1A-1A1A/0-0/0-0/C.</ab>
+                                                                      
+                                    <ab type="pricking"> Primary pricks are visible.</ab>
```

Comment:
Just to make things more beautiful, you could avoid the space before the first character of the content